core/src/main/java/no/nav/dokdistdpi/consumer/dpi/client/DpiClient.java

```diff
@@ -92,42 +92,42 @@ public List<ForsendelseStatusResponse> sendDpiForsendelse(MultipartBodyBuilder m
 			ResponseEntity<String> response = restTemplate.exchange(uri, POST, httpEntity, String.class);
 
 			if (!CREATED.equals(response.getStatusCode())) {
-				log.error(LOG_FEIL_MELDING, response.getStatusCode(), forsendelse.getBestillingsId(), response.getBody());
-				throw new KunneIkkeDistribuereForsendelseException(format(EXCEPTION_FEIL_MELDING, response.getStatusCode(), forsendelse.getBestillingsId(), response.getBody()));
+				log.error(LOG_FEIL_MELDING, response.getStatusCode(), forsendelse.getKonversasjonId(), response.getBody());
```

Comment:
```suggestion
				log.error(LOG_FEIL_MELDING, response.getStatusCode(), konversasjonId, response.getBody());
```
Foreslår å skille ut `forsendelse.getKonversasjonId()` til en egen variabel (blir brukt mange steder i metoden)

Reply:
La til egen variable.